packages/labs/image/src/loaders/sharp.ts

```diff
@@ -0,0 +1,132 @@
+import sharp from 'sharp';
+import { isAspectRatioString, isOutputFormat } from '../types.js';
+import type { ImageProps, OutputFormat, SSRImageService } from '../types.js';
+
+function createService(): SSRImageService {
```

Comment:
An example implementation for an SSR service.  Third-party libraries just need to provide this API, the integration handles supporting `build` vs `dev` and SSG vs SSR